Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
 gem "solidus", github: "solidusio/solidus", branch: branch
 gem "solidus_auth_devise", github: "solidusio/solidus_auth_devise"
+gem "jquery-bar-rating", git: "https://github.com/denkungsart/jquery-bar-rating.git"
```

Comment:
Please don't load the javascript via a gem; vendor it into the assets instead. Then stores can use their own rating plugin in the frontend if they want. 

Also, Gemfiles don't get loaded from engines. Dependencies have to be defined in the gemspec.